README.md

```diff
@@ -1,2 +1,41 @@
-# awesome-books
-Creating a book library with Charles 
+# Awesome books
+
+We are creating a book library with Charles & Patricia as part of Microverse project (week 6)
+
+## Description
+
+"Awesome books" is a simple website that displays a list of books and allows you to add and remove books from that list.
+By building this application, you will learn how to manage data using JavaScript. Thanks to that your website will be more interactive.
+You will also use a medium-fidelity wireframe to build the UI.
+
+
+### How we built the "Awesome books" website
+
+We start by building the core functionalities and ignoring how your website looks. 
+At the very beginning our website will look similar to the UI below. 
+Note that it is plain HTML with no styling, but it will allow you to add and remove books from the list!
+
+<p align="center">
+  <img src="https://github.com/microverseinc/curriculum-javascript/blob/main/books/images/awesome_books_basic_ui.png" alt="Basic UI" width="300px" />
+</p>
+
+Once we have the code working, we will style the core elements: the books list and the form for adding a new book. We will need to adhere to the layout presented in the wireframe, but we will choose your application's styling. So your initial ugly HTML will turn into something similar to this:
```

Comment:
- [ ] Great effort on explaining the details of your project 👏 however what the previous CR meant and according to the general Microverse requirements, you should use [the template](https://github.com/microverseinc/readme-template) provided by Microverse. This will make your project look more professional and it will make it easier for people to set up the project and to reach out to you in the future if they are interested.

Reply:
Thanks @mikethreels for pointing out the template, I used it to make the necessary changes of the project. 
We can still improve it for tomorrow submission if necessary